rc_simulator/radiation.py

```diff
@@ -28,28 +28,66 @@
 class Location(object):
     """Set the Location of the Simulation with an Energy Plus Weather File"""
 
-    def __init__(self, epwfile_path):
-
-        # Set EPW Labels and import epw file
-        epw_labels = ['year', 'month', 'day', 'hour', 'minute', 'datasource', 'drybulb_C', 'dewpoint_C', 'relhum_percent',
-                      'atmos_Pa', 'exthorrad_Whm2', 'extdirrad_Whm2', 'horirsky_Whm2', 'glohorrad_Whm2',
-                      'dirnorrad_Whm2', 'difhorrad_Whm2', 'glohorillum_lux', 'dirnorillum_lux', 'difhorillum_lux',
-                      'zenlum_lux', 'winddir_deg', 'windspd_ms', 'totskycvr_tenths', 'opaqskycvr_tenths', 'visibility_km',
-                      'ceiling_hgt_m', 'presweathobs', 'presweathcodes', 'precip_wtr_mm', 'aerosol_opt_thousandths',
-                      'snowdepth_cm', 'days_last_snow', 'Albedo', 'liq_precip_depth_mm', 'liq_precip_rate_Hour']
-
-        # Import EPW file
-        self.weather_data = pd.read_csv(
-            epwfile_path, skiprows=8, header=None, names=epw_labels).drop('datasource', axis=1)
+    # the location and name should be parsible from the EPW file
+    latitude_deg = None
+    longitude_deg = None
+    name = None
+
+    def __init__(self, path, dtype='epw'):
+
+        if dtype == 'epw':  # ewp weather data file
+
+            # Set EPW Labels and import epw file
+            epw_labels = ['year', 'month', 'day', 'hour', 'minute', 'datasource', 'drybulb_C', 'dewpoint_C',
+                          'relhum_percent', 'atmos_Pa', 'exthorrad_Whm2', 'extdirrad_Whm2', 'horirsky_Whm2',
+                          'glohorrad_Whm2', 'dirnorrad_Whm2', 'difhorrad_Whm2', 'glohorillum_lux',
+                          'dirnorillum_lux', 'difhorillum_lux', 'zenlum_lux', 'winddir_deg', 'windspd_ms',
+                          'totskycvr_tenths', 'opaqskycvr_tenths', 'visibility_km', 'ceiling_hgt_m',
+                          'presweathobs', 'presweathcodes', 'precip_wtr_mm', 'aerosol_opt_thousandths',
+                          'snowdepth_cm', 'days_last_snow', 'Albedo', 'liq_precip_depth_mm',
+                          'liq_precip_rate_Hour']
+
+            # Import EPW file
```

Comment:
I wonder whether it would also be possible to parse the year? Don't worry if you don't have time. It might be something I might look at later :)